src/features/utils/ansibleCfg.ts

```diff
@@ -0,0 +1,82 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import untildify from 'untildify';
+import * as ini from 'ini';
+
+// Get rootPath based on multi-workspace API
+export function getRootPath(editorDocumentUri: vscode.Uri): string | undefined {
+  let rootPath: string | undefined = undefined;
+
+  if (!!vscode.workspace.workspaceFolders) {
+    rootPath = vscode.workspace.workspaceFolders?.[0]?.name;
+  }
+
+  if (typeof vscode.workspace.getWorkspaceFolder === 'function') {
+    const workspaceFolder =
+      vscode.workspace.getWorkspaceFolder(editorDocumentUri);
+
+    rootPath = workspaceFolder?.uri?.path;
+  }
+
+  return rootPath;
```

Comment:
If I understand the intent behind this correctly, it could be simplified even more:
```suggestion
  if (typeof vscode.workspace.getWorkspaceFolder !== 'function') {
      return vscode.workspace.workspaceFolders?.[0]?.name;
  }

  return vscode.workspace.getWorkspaceFolder(editorDocumentUri)?.uri?.path;
```